gordo_components/model/factories/lstm_autoencoder.py

```diff
@@ -71,17 +74,13 @@ class (e.x. Adam(lr=0.01,beta_1=0.9, beta_2=0.999)).  If no arguments are
     """
     n_features_out = n_features_out or n_features
 
-    if len(encoding_dim) != len(encoding_func):
-        raise ValueError(
-            f"Number of layers ({len(encoding_dim)}) and number of functions ({len(encoding_func)}) "
-            f"must be equal for the encoder."
-        )
+    encoding_dim, encoding_func = set_dim_func(
+        encoding_dim, encoding_func, (256, 128, 64), ("tanh", "tanh", "tanh")
```

Comment:
Rather than having the defaults down here, why not have them set in the function signature?

Reply:
Yeah, this didn't feel great. This was done to address the comment mentioned [here](https://github.com/equinor/gordo-components/issues/130#issuecomment-467005835). TL;DR Passing only dim or func with a different number of layers from the default will break. 

My solution here was to first check this and throw a warning if only one is passed, set the default to the one that wasn't passed, and then allow it to break as before if they (the one passed and the default used for the other) are of unequal length.

Reply:
Going to roll this back and keep them in the signature with a less descript error if one not passed. :+1: